Unable to compile a build after #19106

**Godot version:**
<!-- Specify commit hash if non-official. -->
Godot 3.1 master

**OS/device including version:**
Arch Linux, Kernel 4.17.14, clang 6.0.1, pulseaudio 12.2


**Issue description:**
<!-- What happened, and what was expected. -->
Since #19106 was merged, I can not compile Godot 3.1 master branch anymore and get the following error message during the build process:

```
[Initial build] Compiling ==> drivers/gles3/rasterizer_storage_gles3.cpp
Ranlib Library         ==> modules/libmodules.x11.opt.tools.64.llvm.a
drivers/pulseaudio/audio_driver_pulseaudio.cpp:376:43: error: arithmetic on a pointer to void
                                ret = pa_stream_write(ad->pa_str, ptr + write_ofs, bytes_to_write, NULL, 0LL, PA_SEEK_RELATIVE);
                                                                  ~~~ ^
1 error generated.
[Initial build] scons: *** [drivers/pulseaudio/audio_driver_pulseaudio.x11.opt.tools.64.llvm.o] Error 1
Compiling ==> drivers/gles3/shader_compiler_gles3.cpp
scons: building terminated because of errors.


```

--- a/drivers/pulseaudio/audio_driver_pulseaudio.cpp
+++ b/drivers/pulseaudio/audio_driver_pulseaudio.cpp
@@ -349,7 +349,7 @@ void AudioDriverPulseAudio::thread_func(void *p_udata) {
 						}
 						uint32_t l = ad->samples_in[in_idx++];
 						uint32_t r = ad->samples_in[in_idx++];
-						ad->samples_out.write[out_idx++] = (l >> 1 + r >> 1) >> 16;
+						ad->samples_out.write[out_idx++] = ((l >> 1) + (r >> 1)) >> 16;
 					}
 				}
 			}
@@ -373,7 +373,7 @@ void AudioDriverPulseAudio::thread_func(void *p_udata) {
 			if (bytes > 0) {
 				size_t bytes_to_write = MIN(bytes, avail_bytes);
 				const void *ptr = ad->samples_out.ptr();
-				ret = pa_stream_write(ad->pa_str, ptr + write_ofs, bytes_to_write, NULL, 0LL, PA_SEEK_RELATIVE);
+				ret = pa_stream_write(ad->pa_str, (char *)ptr + write_ofs, bytes_to_write, NULL, 0LL, PA_SEEK_RELATIVE);
 				if (ret != 0) {
 					ERR_PRINT("pa_stream_write error");
 				} else {

